From ce24974eceea9b03f77a68489f0a94ee113dc1ed Mon Sep 17 00:00:00 2001
From: wwc <wwcphy@terpmail.umd.edu>
Date: Mon, 2 Jul 2018 16:30:14 -0400
Subject: [PATCH 3/4] fix bindings #1

---
 bem/fastlap.pyx | 34 ++++++++++++++++------------------
 1 file changed, 16 insertions(+), 18 deletions(-)

diff --git a/bem/fastlap.pyx b/bem/fastlap.pyx
index cb217ae..347b07d 100644
--- a/bem/fastlap.pyx
+++ b/bem/fastlap.pyx
@@ -38,24 +38,22 @@ cdef extern from "fastlap.h":
             double *pxf, double *pxnrm,
             int *pnumLev, int *pnumMom, int *pmaxItr,
             double *ptol, int *pjob, double* pAreas) nogil
-
-    cdef enum PANEL_TYPES:
-        FL_POINT "POINT" = 1
-        FL_TRIANGLE "TRIANGLE" = 3
-        FL_QUADRILAT "QUADRILAT" = 4
-
-    cdef enum SOURCE_TYPES:
-        FL_NO_SOURCE "NO_SOURCE" = 0
-        FL_POINT_SOURCE "POINT_SOURCE" = 1
-        FL_CONSTANT_SOURCE "CONSTANT_SOURCE" = 11
-        FL_CONSTANT_DIPOLE "CONSTANT_DIPOLE" = 12
-        FL_LINEAR_SOURCE "LINEAR_SOURCE" = 21
-        FL_LINEAR_DIPOLE "LINEAR_DIPOLE" = 22
-
-    cdef enum JOB_TYPES:
-        FL_FIELD "FIELD" = 0
-        FL_GREEN "GREEN" = 1
-        FL_INDIRECT "INDIRECT" = 2
+ 
+    cdef int FL_POINT "POINT"
+    cdef int FL_TRIANGLE "TRIANGLE"
+    cdef int FL_QUADRILAT "QUADRILAT"
+
+    cdef int FL_NO_SOURCE "NO_SOURCE"
+    cdef int FL_POINT_SOURCE "POINT_SOURCE"
+    cdef int FL_CONSTANT_SOURCE "CONSTANT_SOURCE"
+    cdef int FL_CONSTANT_DIPOLE "CONSTANT_DIPOLE"
+    cdef int FL_LINEAR_SOURCE "LINEAR_SOURCE"
+    cdef int FL_LINEAR_DIPOLE "LINEAR_DIPOLE"
+
+    cdef int FL_FIELD "FIELD"
+    cdef int FL_GREEN "GREEN"
+    cdef int FL_INDIRECT "INDIRECT"
+    
 
 cdef extern from "memtracker.h":
     cdef void mtinit() nogil
-- 
2.7.4

